.buildkite/rebuild_models.yml

```diff
@@ -0,0 +1,43 @@
+# This is a pipeline that rebuilds a model, then uploads the resultant
+# model files and .PDF and other reports as (buildkite, not Julia) artifacts.
+
+steps:
+  - label: ":runner: Dynamically launch Pipelines"
+    plugins:
+      - staticfloat/forerunner:
+          # This will create one job per file
+          watch:
+            - path: benchmarks/.*\.jmd
+          path_processor: per-file
+          target: .buildkite/coalescer/rebuild_models
+          target_type: template
```

Comment:
Can you explain to me what you're trying to do here?

Reply:
I want to watch the `.jmd` files. If there is any modification then run the benchmarks for just that file.

The `.buildkite/coalescer/rebuild_models` currently creates a `MODELS` which has "folder_A/modified-file.jmd" as entries. Currently, the script only does an `echo` (should I do something to pass this) and it is supposed to be used by `rebuild_models.jl` to `weave_file(folder, file)`.

Earlier, something like `!rebuild folder_A` would make it check for the `modified.jmd` and launch a pipeline for each of them.